content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5429,7 +5429,17 @@ const getUsers = await prisma.user.findMany({
 
 ## Client methods
 
-Client-level methods are prefixed by `$`.
+### Remarks
+
+- Client-level methods are prefixed by `$`.
+- `$on` and `$use` do not exist on clients extended using `$extends`
+
+<Admonition type="warning">
+  In extended clients, Client methods do not necessarily exist. If you are extending your client, make sure to check for existence before using Client methods like `$transaction` or `$connect`.
```

Comment:
Why would `$transaction` or `$connect` not exist?

Reply:
I'm not sure I understand the question. In extended clients, all client methods `$transaction` and `$connect` included, either exist or are `undefined`